Makefile

```diff
@@ -2,7 +2,7 @@
 #
 #
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    ?=
```

Comment:
This line basically does nothing (assuming that the tool behaves the same when `SPHINXOPTS` is not set and when `SPHINXOPTS` is set but empty). Still good to have it for documentation purposes I suppose?

Reply:
This is based on the initial and autogenerated Makefile. I've no strong opinion whether to keep it or remove it. Feels a bit silly to have an empty variable though ...